arlpy/uwapm.py

```diff
@@ -784,12 +787,12 @@ def _load_arrivals(self, fname_base):
                                 'rx_depth': [rx_depth[k]],
                                 'rx_range': [rx_range[m]],
                                 'arrival_number': [n],
-                                'arrival_amplitude': [data[0]*_np.exp(1j*data[1])],
-                                'time_of_arrival': [data[2]],
-                                'angle_of_departure': [data[4]],
-                                'angle_of_arrival': [data[5]],
-                                'surface_bounces': [data[6]],
-                                'bottom_bounces': [data[7]]
+                                'arrival_amplitude': [dataA[0]*_np.exp(1j*dataA[1])],
+                                'time_of_arrival': [dataA[2]],
+                                'angle_of_departure': [dataA[4]],
+                                'angle_of_arrival': [dataB[0]],
+                                'surface_bounces': [dataB[1]],
+                                'bottom_bounces': [dataB[2]]
```

Comment:
Was there is a bug that this fixes? If so, could you please elaborate what the bug was to maintain proper log of changes?

Reply:
I removed that change from this pull request and will submit as an issue.